cases/resources/casedb_instance_structure.xml

```diff
@@ -0,0 +1,28 @@
+<instance>
```

Comment:
I need a top level node that isn't ```casedb```. Is ```instance``` a good choice for a name or is there something better?